gbpservice/neutron/services/servicechain/drivers/oneconvergence_intercloud_driver.py

```diff
@@ -213,25 +262,56 @@ def _fetch_template_and_params(self, context, sc_instance,
 
                 config_param_values['RouterId'] = routers[0]['id']
 
+                provider_subnet = self._core_plugin.get_subnet(context._plugin_context, provider_subnet_id)
+
```

Comment:
General approach is for plugin/agent/drivet to retrive fip and not Service Chain driver. Suggest moving the fip call to plugin and not passing it in description field